src/Compilers/Core/Portable/Emit/CommonPEModuleBuilder.cs

```diff
@@ -763,7 +763,7 @@ public ImmutableArray<ISymbolInternal> GetAllMembers()
 
                 if (NestedTypes != null)
                 {
-                    foreach (var type in NestedTypes)
+                    foreach (var type in NestedTypes.OrderBy(t => t.Name))
```

Comment:
:memo: I have not identified a scenario affected by this fix, but seems a good idea to include anyways. It's used in the delta metadatawriter (for EnC).